### PR TITLE
fix: Add suffix to template files

### PR DESCRIPTION
**Pull request by rotu**
Add `.hbs` as a suffix to all [Handlebars](https://handlebarsjs.com/) template files.

This helps in:
- understanding the purpose of files in this repo (especially when searching/grepping)
- troubleshooting when something goes wrong
- prevent spurious tooling warnings, e.g. json syntax errors reported by GitHub and highlighted in red in this screenshot:
    image

### CODE REVIEW

**Comment by wraithgar** [comment]
Look good other than the linting error now that `extname` isn't being used anymore.

File: .github/ISSUE_TEMPLATE/config.yml.handlebars [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file is automatically added by @npmcli/template-oss. Do not edit.
+
+blank_issues_enabled: true
```
Comment (lukekarrys) [inline .github/ISSUE_TEMPLATE/config.yml.handlebars:3]: It looks like this file got created accidentally.

**Comment by lukekarrys** [comment]
Can it use `.hbs` instead of `.handlebars`? I generally prefer shorter file extensions if possible. I think it makes the file sidebar easier to scan visually:

Screenshot 2023-09-15 at 9 04 16 AM
Screenshot 2023-09-15 at 9 04 40 AM

**Comment by rotu** [comment]
> Can it use `.hbs` instead of `.handlebars`? I generally prefer shorter file extensions if possible.

I can do that.

BTW, when doing this PR, I realized that the existing `.yml`/`.md`/... extension is stripped off for purposes of assigning a name to the `partial`. I figure that's why `pkg.json` is named as such (instead of `package.json`, which would probably interfere with tooling).

Since that extension is ignored, maybe we should either:
- Must name the files e.g. `_step-audit.handlebars` and add a comment at the top like `{{! compiled to yaml }}`
- Make the destination format part of the name like `_step-audit-yaml.handlebars` and use `{{> stepAuditYaml }}`

Thoughts?

**Comment by wraithgar** [comment]
ircc think it's called pkg.json so that npm doesn't pick it up and assume that subdirectory is a package.  The presence of a file named `package.json` is very load bearing in npm.

**Comment by wraithgar** [comment]
> Thoughts?

Having the eventual extension be in the filename already is VERY helpful imo, a yaml/etc comment just isn't the same.

**Comment by rotu** [comment]
> ircc think it's called pkg.json so that npm doesn't pick it up and assume that subdirectory is a package. The presence of a file named `package.json` is very load bearing in npm.

Exactly what I meant! Calling a file `package.json` has unintended consequences. But calling it `pkg.json`, `package.json.handlebars`, or `package-json.handlebars` has no such baggage.

> So `_step-audit-yaml.handlebars` would be just keen imo.

Sorry, I don't speak jive. I *think* you mean that I should go with `_step-audit-yaml.handebars` (or, more likely, `_step-audit-yaml.hbs`). Please correct if wrong!

**Comment by wraithgar** [comment]
I think `_step-audit-yaml.hbs` is a good choice, it keeps the `yaml` in the filename while removing `.yaml` from the "extension" chain.

File: .github/ISSUE_TEMPLATE/config.yml.handlebars (hunk 1 above)
Comment (rotu) [reply .github/ISSUE_TEMPLATE/config.yml.handlebars:3]: good catch! oops!

**Comment by wraithgar** [comment]
lgtm, I'll let @lukekarrys land this one.

I really love being able to remove the local eslint config!

**Review by lukekarrys — APPROVED** [review]

**Pull request merged**